packages/babel/src/transformation/file/options/config.json

```diff
@@ -240,6 +236,12 @@
     "description": "set `sources[0]` on returned source map"
   },
 
+  "inputSourceMap": {
+    "type": "boolean",
+    "description": "use existing source map for source map generation - only works with inline source maps",
```

Comment:
@ericvw I'm not sure what you mean. Why do you think this change is necessary and why do you think it will fix the broken source map merging behaviour?